pkg/compose/publish.go

```diff
@@ -87,15 +87,6 @@ func (s *composeService) publish(ctx context.Context, project *types.Project, re
 		if err != nil {
 			return err
 		}
```

Comment:
IHMO if we need to remove one of the 2 identical if, we should keep the one sending the event and remove this one.

Reply:
Fair enough, swapped it around.